gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/CourseGradeOverridePanel.java

```diff
@@ -195,5 +205,36 @@ private String formatPoints(final CourseGrade courseGrade, final Gradebook grade
 		return rval;
 
 	}
+	
+	private String getGradeFromNumber(String newGrade, Map<String, Double> schema)
+	{
+		//Returns the scale whose value equals to the numeric value received, or if it doesn't exists, the highest value lower.
+		Double currentGradeValue = new Double(0.0);
+		try
+		{
+			Double dValue = Double.parseDouble(newGrade.replaceAll(",",".")); //The decimal separator is not relevant here.
```

Comment:
For some reason when I see this comma/decimal replacing going on I cringe!

Shouldn't we just be using DecimalFormat? when dealing with input from other locales vs the string replacement?